google/cloud/storage/BUILD.bazel

```diff
@@ -162,6 +162,7 @@ cc_library(
         ":google_cloud_cpp_storage",
         ":google_cloud_cpp_storage_grpc",
         "//:common",
+        "//google/cloud:google_cloud_cpp_mocks",
```

Comment:
> upstream fix for missing copybara deps

Given that `BUILD.bazel` isn't used downstream, it is unclear what effect this is supposed to have.

Reply:
That's fair....